submissions/recipe_oriented_factorio_life/recipe_oriented_factorio_life.md

```diff
@@ -0,0 +1,75 @@
+
+The chosen rail grid in my practice is a two-way left-hand traffic railroad in order to place the rail signals between the rails. Junctions are straight and left turn only for space-saving. Most stations have a train stop limit of 2 to ensure that a maximum of one train waits outside the station. Sections occupied by a handful of trains waiting outside the stations can be detoured through the fully interconnected grid.
+
+![rail grid](figure/rail_grid.jpg)
+_Only the center grid will build a module, the rest grids are used for traffic and solar paving_
+
+The smallest 2 car trains with 1 locomotive and 1 cargo/fluid wagon were chosen to reduce the size and eliminate the need to balance loading and unloading between wagons. Fuel replenishment for the locomotives set in the LTN's depot stations where trains return to after each delivery.
+
+![rail yard module](figure/rail_yard_module.jpg)
+_Fuel used for locomotives replenishment are also acquired via LTN._
+
+### Module design
+
+Let's start with the simplest example, the iron gear wheel, a recipe with a single ingredient and a single product. A logistic train stop at the bottom of the grid is requesting 8000 iron plates, and a logistic train stop at the top of the grid is providing items (i.e. gears) from the connected warehouse.
+
+![Gear module](figure/rofl_gear_module.jpg)
+_Simply gear module. Raw material station is below, product station is above_
+
+Since the inputs and outputs within this module are fulfilled by trains, our modification and upgrade of the production line within the module will not affect other modules.
+
+![Alt gear module](figure/rofl_gear_module_2.jpg)
+_Gear module with beacons. No change of stations, only the production line between stations._
+
+Next, let‘s look at an advanced circuit module with three types of raw materials. By setting the logistic train stop requesting multiple ingredients, we were able to unload all of our raw materials at a single station. At this point all the raw materials are mixed in one warehouse， we then use filter miniloaders (similar to filter inserters) to separate each material onto a separate belt to supply the assembly machine.
```

Comment:
Maybe expand on what warehouses and loaders are for anyone not familiar with the concepts. This could also be the point to work in the link to the loader and warehouse mods. Also probably worth saying that is can be done without these mods - possibly could use a compare image thing to show a version using the mod and one not using the mods - see https://alt-f4.blog/ALTF4-X/#compare-images

Reply:
Further explanation of such popular mods as Warehouse and Loader may be a bit redundant to me, especially since such mods have appeared in Alt-F4 in the past. 

Adding a vanilla compare to a modded loading station is a fascinating idea, it's just that I'm currently having trouble making relevant media content and I need help to make pictures.

Reply:
After rethinking, I realized that it was impossible to simply and elegantly reproduce a station with similar functionality in vanilla. The main difficulty is that LTN can control the quantity of different raw material items and be able to unload multiple types of items in one station without stoppage. Vanilla, on the other hand, cannot distinguish different items even with the 1.1 train stop limit, and there is a high risk of an overflow of one ingredient and jams the traffic.